sample-app/src/main/java/com/squareup/contour/sample/ExpandableBioCard1.kt

```diff
@@ -33,10 +34,15 @@ import com.squareup.contour.ContourLayout
 import com.squareup.picasso.Picasso
 
 @SuppressLint("SetTextI18n")
-class ExpandableBioCard1(context: Context) : ContourLayout(context) {
+class ExpandableBioCard1(context: Context, attributeSet: AttributeSet? = null) : ContourLayout(context) {
   private val avatar = CircularImageView(context).apply {
     scaleType = CENTER_CROP
-    Picasso.get().load("https://upload.wikimedia.org/wikipedia/en/9/92/BenSisko.jpg").into(this)
+
+    //image will appear on emulator and devices
+    if (!isInEditMode)
+    Picasso.get()
+            .load("https://upload.wikimedia.org/wikipedia/en/9/92/BenSisko.jpg")
+            .into(this)
```

Comment:
Can you revert this formatting change?

Reply:
Resolving this in favor of https://github.com/cashapp/contour/pull/110#discussion_r496350121